jenkins/Jenkinsfile

```diff
@@ -193,7 +195,7 @@ pipeline {
 
     stage('Pull Leonardo') {
         steps {
-            git credentialsId: 'jenkins-ssh-github', url: 'git@github.com:DataBiosphere/leonardo.git', branch: 'rt-add-algebra-2'  // TODO revert me
+            git credentialsId: 'jenkins-ssh-github', url: 'git@github.com:DataBiosphere/leonardo.git', branch: 'ky_ia_1765_image_hardening_using_appsec_libraries'  // TODO revert me
```

Comment:
Oops did I leave this by mistake? Reminder to change to `develop` before merge.

Reply:
No worries. It's too easy to forget. I probably will too 😝